internal/testing/e2e/common.go

```diff
@@ -218,6 +218,7 @@ api:
     httpport: "51509"
 
 redis:
+  mutexTimeout: "5m"
```

Comment:
Is it better to decrease 5m value to just 1m?

Reply:
Ok, i'll change it